tools/analyzer_plugin/lib/src/component_usage.dart

```diff
@@ -162,7 +162,7 @@ FluentComponentUsage identifyUsage(AstNode node) {
         return usage;
       }
     }
-    if (node is ArgumentList || node is Statement || node is FunctionBody) {
+    if (node is Statement || node is FunctionBody) {
```

Comment:
Removed check for `ArgumentList` because children in component usages are part of `ArgumentList`s